includes/template-functions.php

```diff
@@ -726,6 +726,26 @@ function sensei_the_module_id() {
 
 }
 
+/**
+ * Gets a count of the lessons in the current module in the modules loop.
+ *
+ * @since 3.1.0
+ *
+ * @return int|false Number of lessons in the current module, or `false` on failure.
+ */
+function sensei_module_lesson_count() {
+	global $sensei_modules_loop;
+
+	if ( ! isset( $sensei_modules_loop['course_id'] ) || ! isset( $sensei_modules_loop['current_module']->term_id ) ) {
+		return false;
```

Comment:
I think it would be better if we always returned an `int` from this function and returned a 0 here instead of `false`.

Also, we're checking that `$sensei_modules_loop['current_module']->term_id` is set, but if `$sensei_modules_loop['current_module']` isn't set this function would still error out.

Reply:
I have tested and `isset( $a['b']->c )` does not produce any errors or notices. If you have time, please test and confirm it. If not, I can add the check for `$sensei_modules_loop['current_module']`

Reply:
If we're adding the `isset` to be safe, then shouldn't we make it extra safe and also check that `$sensei_modules_loop['current_module']` is set? We're already checking `$sensei_modules_loop['course_id'] `, so it seems to make sense to do the same for the module.

Re: 3.1.0, that release is coming out very soon, and this PR is going back and forth a lot, so I'm not sure it will make it in. We can leave it for now, but if it doesn't get wrapped up very soon it's going to get bumped.

Reply:
Ok. I have updated the PR. Please check it. Thanks.